vlib/v/checker/tests/import_unused_warning.out

```diff
@@ -1,2 +1,5 @@
-`vlib/v/checker/tests/import_unused_warning.v` warning: the following imports were never used:
- * time
+./vlib/v/checker/tests/import_unused_warning.v:1:8: warning: module 'time' is imported but never used 
```

Comment:
remove the `./` at the start, so that the test works

Reply:
Thank's @spytheman .... sorry, I just pay my wifi bills... :smile: